modules/loop_search_node_settings/loop_search_node_settings.features.inc

```diff
@@ -21,7 +21,6 @@ function loop_search_node_settings_default_search_api_index() {
           "loop_documents_collection",
           "external_sources",
           "instruction",
-          "leaf",
```

Comment:
Shouldn't this be part of the previous PR (https://github.com/os2loop/profile/pull/113)?